src/main/scala/com/databricks/spark/sql/perf/mllib/feature/Tokenizer.scala

```diff
@@ -0,0 +1,28 @@
+package com.databricks.spark.sql.perf.mllib.feature
+
+import org.apache.spark.ml
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.ml.PipelineStage
+import org.apache.spark.sql._
+
+import com.databricks.spark.sql.perf.mllib.OptionImplicits._
+import com.databricks.spark.sql.perf.mllib.data.DataGenerator
+import com.databricks.spark.sql.perf.mllib.{BenchmarkAlgorithm, MLBenchContext, TestFromTraining}
+
+/** Object for testing Tokenizer performance */
+object Tokenizer extends BenchmarkAlgorithm with TestFromTraining with UnaryTransformer {
+
+  override def trainingDataSet(ctx: MLBenchContext): DataFrame = {
+    import ctx.params._
+    import ctx.sqlContext.implicits._
+
+    DataGenerator.generateRandString(ctx.sqlContext,
+      numExamples, ctx.seed(), numPartitions, 100, 200, "abcdefg ")
+      .select($"label".as(inputCol))
```

Comment:
@WeichenXu123  I didn't add any data generation code for LDA, but I did refactor the `ldaNumDocs` and `ldaNumVocabulary` params into `numDocs` and `numVocabulary`, respectively; see my `ml-experiment-newtests` branch here:

https://github.com/smurching/spark-sql-perf/tree/ml-experiment-newtests